app/mailers/prison_mailer.rb

```diff
@@ -33,4 +37,17 @@ def rejected(visit)
            prisoner: visit.prisoner_full_name
          )
   end
+
+private
+
+  def do_not_send_to_prison
+    message.to = visit.contact_email_address
+    message.delivery_method.settings.
+      merge!(Rails.configuration.action_mailer.smtp_settings)
```

Comment:
Is it correct that `action_mailer.smtp_settings` has precedence over the message's settings?